### PR TITLE
Update asset guide

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- Updates the asset guide to show yml as an example
- Updates to the latest version of PagerDuty asset

To do:
- [x] Get SHA for PagerDuty asset
- [x] Copy to all versions

### CODE REVIEW

File: content/sensu-go/5.0/guides/install-check-executables-with-assets.md [hunk 1]
```diff
@@ -11,153 +11,117 @@ menu:
...
-{{< highlight json >}}
-{"component":"pipelined","namespace":"default","handler":"influx-db","level":"info","msg":"event pipe handler executed","output":"metric sent to influx-db","status":0,"time":"2018-10-16T13:17:33-07:00"}
-{{< /highlight >}}
-
-## How to create a mutator that depends on an asset 
-
-### Adding an asset to a mutator on creation
 
-[Create an asset][2] (ex. `transformer`), and create a mutator (ex. `transform-metrics`).
+Save the definition to a file, and add to Sensu using sensuctl.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/guides/install-check-executables-with-assets.md]:
```suggestion
Save the definition to a file, e.g. `pagerduty-handler.json`, and add to Sensu using sensuctl.
```

File: content/sensu-go/5.0/guides/install-check-executables-with-assets.md [hunk 2]
```diff
@@ -11,153 +11,117 @@ menu:
...
 
 {{< highlight shell >}}
-sensuctl mutator create transform-metrics \
---command "transform --type metrics" \
---runtime-assets transformer
+sensuctl create --file filename.json
```
Comment (cwjohnston) [inline content/sensu-go/5.0/guides/install-check-executables-with-assets.md]:
```suggestion
sensuctl create --file pagerduty-handler.json
```

File: content/sensu-go/5.0/guides/install-check-executables-with-assets.md [hunk 3]
```diff
@@ -11,153 +11,117 @@ menu:
...
 {{< /highlight >}}
 
-### Validating the mutator asset
+Now that Sensu can create incidents in PagerDuty, we can automate this workflow by adding the `pagerduty` handler to our Sensu service checks.
+To get started with checks, see the [guide to monitoring server resources](../monitor-server-resources).
 
-Once the mutator is setup, it must be attached to a handler to mutate the event output (ex. `influx-db`).
+You can use sensuctl to see available checks, handlers, and other Sensu resources.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/guides/install-check-executables-with-assets.md]: This feels somewhat disconnected from the rest of what's described here about assets.

Perhaps we move the "to get started with checks" line down to Next steps and get rid of the content about using sensuctl to list available resources?

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Comment by apaskulin** [comment]
@cwjohnston Great feedback. Thank you! I've incorporated your changes in https://github.com/sensu/sensu-docs/pull/1309/commits/41e8747c6772f59d4af7a979ce15fa5e669fcc73. I removed the list commands, but kept the reference to the checks guide as part of the third step.

**Review by cwjohnston — APPROVED** [review]
Thanks, looks good!

**Pull request merged**